pom.xml

```diff
@@ -6,7 +6,7 @@
   <!-- Arifact name and version information -->
   <groupId>net.snowflake</groupId>
   <artifactId>snowflake-ingest-sdk</artifactId>
-  <version>1.1.3</version>
+  <version>1.1.4-SNAPSHOT</version>
```

Comment:
Does this mean we need to manually update the pom everytime there is a new release? Also how does this work with a new commit after this change?

Reply:
There's the maven-release-plugin to help drive this workflow and do these updates automatically: https://maven.apache.org/maven-release/maven-release-plugin/usage.html

Reply:
@sfc-gh-japatel Do we need to update the `DEFAULT_VERSION` as well?

Reply:
Thanks @sfc-gh-pbennes we will look into it.

@sfc-gh-tzhang I think it would be worth making DEFAULT_VERSION to a snapshot as well. The only place where we are using this is to send telemetry info to server side and it might help us to debug further. 

If we want to change DEFAULT_VERSION to have suffix as well, it might be worth to sticking to current approach. Because I am not sure if the plugin (which automatically prepares and releases and modifies the pom) can handle modification in JAVA files.

Reply:
@sfc-gh-tzhang 
Snapshot_deploy.sh file will deploy the latest code to [snapshot repo](https://nexus.int.snowflakecomputing.com/repository/Snapshots/) every night with help of [jenkins job](https://ci-dev-122.int.snowflakecomputing.com/job/MavenPushSnowpipeJavaSDKSnapshot/).
Benchmark framework will grab the latest build from snapshot repo and run the tests against the latest master(deployed to snapshot repo)

Reply:
@sfc-gh-dshah we will need to change this to 1.1.5-snapshot, I released 1.1.4 yesterday

Reply:
also lets modify `DEFAULT_VERSION` as toby mentioned. It's in java file for us to pass in version from client side to server side. .

Reply:
@sfc-gh-japatel Should I change the version [here](https://github.com/snowflakedb/snowflake-ingest-java/blob/master/src/main/java/net/snowflake/ingest/connection/RequestBuilder.java#L130) ?

Reply:
Yes..

Reply:
test is failing: `testUserAgentSuffixForInsertFileAPI` it's a required merge gate.